CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.15)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Why decrease the version here?

Reply:
This is for compatibility with platforms which currently don't supply v 3.15.

Reply:
Got it